.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .sass-cache
 .jekyll-cache
 .jekyll-metadata
+js/node_modules/
```

Comment:
I think you might have run `npm install` from within the `js` directory rather than the repository root, which is the general practice. I'd suggest gitignoring `node-modules/` and running `npm install` from the root.